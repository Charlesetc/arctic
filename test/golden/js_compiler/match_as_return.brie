define main [
  match X [ X -> 2 ]
]
