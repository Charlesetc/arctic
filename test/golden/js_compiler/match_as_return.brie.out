const global_functions = {1: {}};
function main() {
	return (function(_matched_target){
	if (_matched_got =  _matched_target.X) {
	return 2;

	}
}({X: []}));
}
var test/golden/js_compiler/match_as_return.brie_main = new_closure(1)
console.log(main());